stdlib/test/builtin/test_list.mojo

```diff
@@ -39,7 +39,47 @@ fn test_repr_list() raises:
     assert_equal(empty.__repr__(), "[]")
 
 
+fn test_contains() raises:
+    # List
```

Comment:
**Suggestion** You can drop these list specific tests since they're tested elsewhere